sdk/core/azure-core-amqp/CMakeLists.txt

```diff
@@ -160,25 +152,111 @@ set(AZURE_CORE_AMQP_SOURCE
     src/models/private/source_impl.hpp
     src/models/private/target_impl.hpp
     src/models/private/value_impl.hpp
-    src/network/amqp_header_transport.cpp
-    src/network/private/transport_impl.hpp
-    src/network/private/transport_impl.hpp
-    src/network/sasl_transport.cpp
-    src/network/socket_listener.cpp
-    src/network/socket_transport.cpp
-    src/network/tls_transport.cpp
-    src/network/transport.cpp
     src/private/package_version.hpp
 )
 
 if (USE_UAMQP)
-add_library(azure-core-amqp ${AZURE_CORE_AMQP_SOURCE} ${AZURE_CORE_AMQP_HEADER} $<TARGET_OBJECTS:uamqp>)
+set(AZURE_UAMQP_SOURCE
+  src/impl/uamqp/amqp/cancellable.cpp
+  src/impl/uamqp/amqp/claim_based_security.cpp
+  src/impl/uamqp/amqp/connection.cpp
+  src/impl/uamqp/amqp/connection_string_credential.cpp
+  src/impl/uamqp/amqp/link.cpp
+  src/impl/uamqp/amqp/management.cpp
+  src/impl/uamqp/amqp/message_receiver.cpp
+  src/impl/uamqp/amqp/message_sender.cpp
+  src/impl/uamqp/amqp/session.cpp
+  src/impl/uamqp/amqp/private/claims_based_security_impl.hpp
+  src/impl/uamqp/amqp/private/connection_impl.hpp
+  src/impl/uamqp/amqp/private/link_impl.hpp
+  src/impl/uamqp/amqp/private/management_impl.hpp
+  src/impl/uamqp/amqp/private/message_receiver_impl.hpp
+  src/impl/uamqp/amqp/private/message_sender_impl.hpp
+  src/impl/uamqp/amqp/private/session_impl.hpp
+#  src/impl/uamqp/models/amqp_detach.cpp
+#  src/impl/uamqp/models/amqp_error.cpp
+#  src/impl/uamqp/models/amqp_header.cpp
+#  src/impl/uamqp/models/amqp_message.cpp
+#  src/impl/uamqp/models/amqp_properties.cpp
+#  src/impl/uamqp/models/amqp_transfer.cpp
+#  src/impl/uamqp/models/amqp_value.cpp
```

Comment:
Why are some of these commented out?

Reply:
Because they shouldn't be there - now. But that may change in the future. I haven't made up my mind about how much I should be forking the implementations.